src/i965_gpe_utils.c

```diff
@@ -2751,7 +2755,7 @@ gen9_add_buffer_2d_gpe_surface(VADriverContextP ctx,
     gpe_surface.cacheability_control = i965->intel.mocs_state;
     gpe_surface.format = format;
 
-    gen9_gpe_context_add_surface(gpe_context, &gpe_surface, index);
+    gpe->context_add_surface(gpe_context, &gpe_surface, index);
 }
```

Comment:
Can I use i965 as the prefix of function name? The function only apply to gen8/9 and not apply to gen7 and before.